.github/workflows/arise-deploy.yml

```
@@ -44,7 +44,7 @@ jobs:
         run: sudo apt-get install -y pandoc
 
       - name: Build Arise
-        run: bash arise build
+        run: bash arise
```

rejecting breaking change to invocation syntax
```suggestion
        run: bash arise build
```